mesonbuild/compilers/cuda.py

```diff
@@ -309,6 +309,7 @@ def get_xcompiler_val(flag: str, flagit: T.Iterator[str]) -> str:
                 raise ValueError("-Xcompiler flag merging failed, unknown argument form!")
         return xflags
 
+    @classmethod
     def _to_host_flags(self, flags: T.List[str], phase: _Phase = _Phase.COMPILER) -> T.List[str]:
```

Comment:
If you're changing this to a classmethod it should take cls instead of self.

Luckily the other 2 CudaCompiler methods called in this function seem to already be classmethods as well. But invoking them as `cls._xxx()` rather than `self._xxx()` makes this a lot clearer.